app/views/map_assets/create.js.erb

```diff
@@ -1,6 +1,7 @@
 <% if @map_asset.errors.any? %>
   $("#new_asset_form").html("<%= j render('form', map_asset: @map_asset) %>");
+  rating_binder();
 <% else %>
   $("#assets_table").prepend("<%= j render('map_asset_row', asset: @map_asset) %>");
-  $(".close-modal").click();
+  $(".close-modal").click()
```

Comment:
the semi-colon was fine, you should not remove it.

Reply:
Okay, Fixed